### ISSUE TITLE
Regions not played

### ISSUE THREAD

**Issue by dwuertz** [sfztools/sfizz#777]
With current master, the following instrument is not behaving correctly anymore

```
<region>
key=60 sample=break.wav offset=0 end=7535 group=1 off_by=1

<region>
key=61 sample=break.wav offset=7535 end=14413 group=1 off_by=1

```
Only first region (key=60) is playing, second isn't.
The bug was introduced with commit 10c5e64b

**Comment by paulfd** [comment]
I probably corrected this just now in #778 because I stumbled on a potential issue with this. Let me merge it later and you can tell me if it helps.

**Comment by dwuertz** [comment]
Thanks @paulfd, I can confirm that #778 fixes the problem!

**Comment by paulfd** [comment]
Good to hear ;)

**Comment by paulfd** [comment]
Merged #778, thanks for the report !
